pkg/types/v1/config.go

```diff
@@ -146,7 +146,7 @@ func (r *RunConfig) Sanitize() error {
 
 // InstallSpec struct represents all the installation action details
 type InstallSpec struct {
-	Target          string              `yaml:"target,omitempty" mapstructure:"target"`
+	Target          string              `yaml:"device,omitempty" mapstructure:"device"`
```

Comment:
Is this a "breaking" change? Is it something we should warn the users about? (e.g. if they have configs that used `target`).

Reply:
not really, this is the target in the elemental config, which now was unused by agent as it had its own target.

What this does it to sync it with the agent so it can load it from the cloud config directly. This is ONLY used on the grub install IIRC and we set it manually with the data from the agent Config already. This just allows us to have it in sync automatically :D